language/automl/automl_natural_language_dataset.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to perform basic operations on Dataset
+with the Google AutoML Natural Language API.
+
+For more information, see the tutorial page at
+https://cloud.google.com/natural-language/automl/docs/
+"""
+
+# [START automl_natural_language_import]
+import argparse
+import os
+
+from google.cloud import automl_v1beta1 as automl
+
+# [END automl_natural_language_import]
+
+
+# [START automl_natural_language_create_dataset]
+def create_dataset(project_id, compute_region, dataset_name, multilabel=False):
+    """Create a dataset.
+    Args:
+        project_id: Id of the project.
+        compute_region: Region name.
+        dataset_name: Name of the dataset.
+        multilabel: Type of the classification problem.
+            False - MULTICLASS, True - MULTILABEL.
+            Default is False.
+    """
+    client = automl.AutoMlClient()
+
+    # A resource that represents Google Cloud Platform location.
+    project_location = client.location_path(project_id, compute_region)
+
+    # Classification type is assigned based on multilabel value.
+    classification_type = "MULTICLASS"
+    if multilabel:
+        classification_type = "MULTILABEL"
+
+    # Specify the text classification type for the dataset.
+    dataset_metadata = {"classification_type": classification_type}
+
+    # Set dataset name and metadata.
+    my_dataset = {
+        "display_name": dataset_name,
+        "text_classification_dataset_metadata": dataset_metadata,
+    }
+
+    # Create a dataset with the dataset metadata in the region.
+    dataset = client.create_dataset(project_location, my_dataset)
+
+    # Display the dataset information.
+    print("Dataset name: {}".format(dataset.name))
+    print("Dataset id: {}".format(dataset.name.split("/")[-1]))
+    print("Dataset display name: {}".format(dataset.display_name))
+    print("Text classification dataset metadata:")
+    print("\t{}".format(dataset.text_classification_dataset_metadata))
+    print("Dataset example count: {}".format(dataset.example_count))
+    print("Dataset create time:")
+    print("\tseconds: {}".format(dataset.create_time.seconds))
+    print("\tnanos: {}".format(dataset.create_time.nanos))
+
+
+# [END automl_natural_language_create_dataset]
+
+
+# [START automl_natural_language_list_datasets]

```

Comment:
Same as above comments. Region tag should be inside the function body.

Reply:
done